west.yml

```diff
@@ -103,6 +103,9 @@ manifest:
     - name: mipi-sys-t
       path: modules/debug/mipi-sys-t
       revision: baf51863f19f009b92e762115ba5572a5b996b92
+    - name: hal_xtensa
+      revision: d4c49675b5b49e66ae69fe8b90d5f5290c63ca2e
```

Comment:
dont you need to be pointing to zephyrproject-rtos/hal_xtensa#1?

Reply:
This should be as https://github.com/zephyrproject-rtos/hal_xtensa/pull/1/commits/d4c49675b5b49e66ae69fe8b90d5f5290c63ca2e is the latest commit there. Once the HAL PR is merged, I will double-check the SHA.

Reply:
You can do revision: pull/1/head, so this PR is testing with the HAL PR.

Reply:
Updated west.yml so CI can run. Will revert once the HAL PR is merged.

Reply:
HAL PR is merged, and the revision has been updated to latest commit from there.